examples/jsm/controls/OrbitControls.js

```diff
@@ -358,6 +359,27 @@ class OrbitControls extends EventDispatcher {
 		const pointers = [];
 		const pointerPositions = {};
 
+
+		function getPosition( obj, vec ) {
+
+			if ( ! obj.parent || obj.parent.type === 'Scene' )
+				return vec.copy( obj.position );
+
+			obj.getWorldPosition( vec );
+
+			return vec;
+
+		}
```

Comment:
I wonder if we could simplify the PR and always use `getWorldPosition()` instead of introducing `getPosition()`. The additional overhead for the default case (no parent) is so small that there is no need for a different code path.

Reply:
I think that's a good point. I'll apply your suggestions later and test again